test/data/remote/services/county_service_test.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:convert';
+
+import 'package:dio/dio.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import 'package:vost/constants.dart';
+import 'package:vost/data/remote/endpoints/county_endpoints.dart';
+import 'package:vost/data/remote/models/response/base_list_response.dart';
+import 'package:vost/data/remote/services/county_service.dart';
+
+import '../../../utils/dioadapter/_base_dio_adapter.dart';
```

Comment:
Same. Or is there any reason behind testing that forces you to use relative paths instead?